static/app/components/seenByList.stories.tsx

```diff
@@ -0,0 +1,151 @@
+import {Fragment, useEffect} from 'react';
+
+import Placeholder from 'sentry/components/placeholder';
+import SeenByList from 'sentry/components/seenByList';
+import JSXProperty from 'sentry/components/stories/jsxProperty';
+import Matrix from 'sentry/components/stories/matrix';
+import SizingWindow from 'sentry/components/stories/sizingWindow';
+import storyBook from 'sentry/stories/storyBook';
+import {useMembers} from 'sentry/utils/useMembers';
+import {useUser} from 'sentry/utils/useUser';
+
+function useLoadedMembers() {
+  const {members, loadMore, ...rest} = useMembers();
+
+  useEffect(() => {
+    // `loadMore` is not referentially stable, so we cannot include it in the dependencies array
+    loadMore();
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  return {members, loadMore, ...rest};
+}
+
+export default storyBook(SeenByList, story => {
+  story('Default', () => {
+    const {members, fetching} = useLoadedMembers();
+
+    return (
+      <SizingWindow display="block" style={{width: '50%'}}>
+        {fetching ? <Placeholder /> : <SeenByList seenBy={members} />}
+      </SizingWindow>
+    );
+  });
+
+  story('iconTooltip', () => {
+    const {members, fetching} = useLoadedMembers();
+
+    return (
+      <Fragment>
+        <p>
+          Default is{' '}
+          <JSXProperty name="iconTooltip" value="People who have viewed this" />
+        </p>
+        <SizingWindow display="block" style={{width: '50%'}}>
+          {fetching ? (
+            <Placeholder />
+          ) : (
+            <SeenByList
+              seenBy={members}
+              iconTooltip="These folks have all seen this record"
+            />
+          )}
+        </SizingWindow>
+      </Fragment>
+    );
+  });
+
+  story('Shows users except yourself', () => {
+    const user = useUser();
+    const {members, fetching} = useLoadedMembers();
+
+    return (
+      <Fragment>
+        <p>
+          In this example we've explicitly put `user` at the start of the list, that's
+          you! But it'll be filtered out. The idea is that that viewer is more interested
```

Comment:
does this mean even if we put `user` at the front it'll never show up?

Reply:
although in user feedback i'm able to see myself in the seenBy list

Reply:
use feedback product uses an <AvatarList> not <SeenByList>. well that's kinda silly.

Reply:
`user` not appearing might relate to the fact that the avatars are reversed?

https://github.com/getsentry/sentry/blob/3c9c712be4ccb07227f49f79c0dbc6e8bc20a538/static/app/components/avatar/avatarList.tsx#L38

Reply:
oh nvm! i see this line https://github.com/getsentry/sentry/blob/bd3193736c784689f8a5ec2b046fd830bd2c395d/static/app/components/seenByList.tsx#L39

Reply:
looks like feedback directly uses `AvatarList` (no filter there) rather than `SeenByList` so that makes sense too

Reply:
AvatarList is more popular in the codebase, good choice!